tests/integration/spark/test_histograms.py

```diff
@@ -0,0 +1,72 @@
+import unittest, PyRDF
+
```

Comment:
Perhaps we can call this file: `test_inv_mass.py`?

Reply:
Ok. Sounds good. I'll change this too 👍